### PR TITLE
Refactor reconciler utils a bit

### PR DESCRIPTION
**Pull request by eseliger**
After the overhaul of syncer spawning, I realized the loadExternalService helper could be simplified in the reconciler as well. This does that and since we don't mock this out by passing in a mock store, we don't need the interfaces in this package anymore at all.

### CODE REVIEW

File: enterprise/internal/campaigns/reconciler/util.go [hunk 1]
```diff
@@ -5,28 +5,14 @@ import (
 
 	"github.com/pkg/errors"
 
-	"github.com/sourcegraph/sourcegraph/internal/api"
 	"github.com/sourcegraph/sourcegraph/internal/database"
 	"github.com/sourcegraph/sourcegraph/internal/types"
 	"github.com/sourcegraph/sourcegraph/schema"
 )
 
 // TODO: copy-pasted
```
Comment (LawnGnome) [inline enterprise/internal/campaigns/reconciler/util.go:13]: _Is_ this still copy-pasted?

**Review by LawnGnome — APPROVED** [review]
Good simplification! It feels weird that we didn't actually need the interfaces there, but since everything passed, I guess we're good. :+1:

File: enterprise/internal/campaigns/reconciler/util.go (hunk 1 above)
Comment (mrnugget) [reply enterprise/internal/campaigns/reconciler/util.go:13]: Always safe to add this: _It wasn't me who wrote this, whatyou looking at me for?_

**Review by mrnugget — APPROVED** [review]

File: enterprise/internal/campaigns/reconciler/util.go (hunk 1 above)
Comment (eseliger) [reply enterprise/internal/campaigns/reconciler/util.go:13]: Yeah it still is, and it was me who copy-pasted it so I deserve this comment 😬 Will be fixed soonish once we have site-tokens and stuff.

Comment (LawnGnome) [reply enterprise/internal/campaigns/reconciler/util.go:13]: Copy/paste away, then! :artist:

**Pull request merged**